app/views/jobs/show.html.erb

```diff
@@ -21,86 +21,102 @@
   <p>It will delete all the data related to it.</p>
 <% end %>
 
-<div class="alert alert-success border-success text-dark">
-  <h4 class="alert-heading">This job is currently <%= @job.status %></h4>
-  <p>
-    It has been running for XX mins.
-    This is some more information about what you can currently do while the job is running,
-    and some ways you may be able to pass your time.
-  </p>
-</div>
+<ul class="list-group list-group-horizontal my-4">
+  <li class="list-group-item">Status: <%= job_status_text(@job) %></li>
+
+  <%- if @job.duration_seconds.present? %>
+    <li class="list-group-item">Duration: <%= ActiveSupport::Duration.build(@job.duration_seconds).inspect %></li>
+  <% end %>
+    <li class="list-group-item">Filesize: <%= number_to_human_size(@job.extraction_folder_size_in_bytes) %></li>
+    <li class="list-group-item">Records Per Page: <%= @extraction_definition.per_page %></li>
+    <li class="list-group-item">Total Pages: <%= @job.documents.total_pages %></li>
+    <li class="list-group-item">Total Records: <%= @extraction_definition.per_page * @job.documents.total_pages %></li>
```

Comment:
That's a bit misleading here, the count won't right if all pages are not filled.

Reply:
Fair call, I'll update it to say that it's an estimate and add a tooltip for how it's generated